docs/modules/user-guide/nav.adoc

```diff
@@ -13,6 +13,7 @@
 **** xref:adding-apply-commands-to-a-devfile.adoc[]
 **** xref:adding-composite-commands-to-a-devfile.adoc[]
 *** xref:adding-components-to-a-devfile.adoc[]
+**** xref:defining-a-top-level-attribute.adoc[]
```

Comment:
Could we say `adding-a-top-level-attribute.adoc[]` I think the verb choice of `adding` instead of `defining` would be more consistent with the other docs it's organized with.